lib/handle-create-event.js

```diff
@@ -26,23 +27,39 @@ async function handleCreateEvent (context) {
     debug(`ignoring: "${ref}" does not match /^first-timers-/`)
     return
   }
+  const customTemplateUrl = config.template
 
   const state = {
     api: context.github,
     debug,
     owner: repository.owner.login,
     repo: repository.name,
     branch: ref,
-    template,
-    labels: config.labels,
+    template: config.template,
+    config: config,
     sha: null,
     repoDefaultBranch: repository.default_branch
   }
 
+  await getTemplateContent(state, customTemplateUrl)
+
   return getBranch(state)
     .then(getCommit.bind(null, state))
     .then(createIssue.bind(null, state))
     .catch((error) => {
       debug(error.toString())
     })
 }
+
+function getTemplateContent (state, path) {
+  return state.api.repos.getContent({
```

Comment:
if `state.customTemplateUrl` is not set, then return the app’s `template` content defined in line 13